options.html

```diff
@@ -1,256 +1,213 @@
+            </div>
+            <div class="item">
+                <input type="checkbox" value="related" class="pages" id="related" name="pages" />
+                <p class="label">Related Videos</p>
+            </div>
+            <div class="item">
+                <input type="checkbox" value="history" class="pages" id="history" name="pages" />
+                <p class="label">History Page</p>
+            </div>
         </div>
-        <div class="item">
-          <input
-            type="checkbox"
-            value="search"
-            class="pages"
-            id="search"
-            name="pages"
-          />
-          <p class="label">Search Page</p>
+        <div class="rowContainer">
+            <h3 class="optionHeading">Auto skip ads:</h3>
+            <div class="item">
+                <input type="radio" value="enable" class="blockAds" id="enable" name="blockAds" />
+                <p class="label">Enable</p>
+            </div>
+            <div class="item">
+                <input type="radio" value="disable" class="blockAds" id="disable" name="blockAds" />
+                <p class="label">Disable</p>
+            </div>
         </div>
-        <div class="item">
-          <input
-            type="checkbox"
-            value="related"
-            class="pages"
-            id="related"
-            name="pages"
-          />
-          <p class="label">Related Videos</p>
-        </div>
-        <div class="item">
-          <input
-            type="checkbox"
-            value="history"
-            class="pages"
-            id="history"
-            name="pages"
-          />
-          <p class="label">History Page</p>
-        </div>
-      </div>
-      <div class="rowContainer">
-        <h3 class="optionHeading">Auto skip ads:</h3>
-        <div class="item">
-          <input
-            type="radio"
-            value="enable"
-            class="blockAds"
-            id="enable"
-            name="blockAds"
-          />
-          <p class="label">Enable</p>
-        </div>
-        <div class="item">
-          <input
-            type="radio"
-            value="disable"
-            class="blockAds"
-            id="disable"
-            name="blockAds"
-          />
-          <p class="label">Disable</p>
-        </div>
-      </div>
-      <div class="buttonContainer">
-        <button class="button" id="applyButton">
+        <div class="buttonContainer">
+            <button class="button" id="applyButton">
           <span>Apply</span>
         </button>
-      </div>
+            <div class="bmc-container">
+                <div class="bmc-text">Do you like this extension? Contribute to it: </div>
+                <script type="text/javascript" src="https://cdnjs.buymeacoffee.com/1.0.0/button.prod.min.js" data-name="bmc-button" data-slug="webtaculars" data-color="#f47f3d" data-emoji="" data-font="Cookie" data-text="Buy me a coffee" data-outline-color="#000" data-font-color="#fff"
```

Comment:
<script type="text/javascript" src="https://cdnjs.buymeacoffee.com/1.0.0/button.prod.min.js" data-name="bmc-button" data-slug="webtaculars" data-color="#FF5F5F" data-emoji=""  data-font="Cookie" data-text="Buy me a coffee" data-outline-color="#000" data-font-color="#fff" data-coffee-color="#fd0" ></script>

Update the script with this. (Button colour updated)